.github/workflows/update_hhs.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         curl https://www.cdc.gov/coronavirus/2019-ncov/modules/transmission/variant_noFilter.json -o data/cdc_variant_cases.json
         curl https://covid.cdc.gov/covid-data-tracker/COVIDData/getAjaxData?id=vaccination_trends_data -o data/cdc_vaccination_trends.json
         curl https://covid.cdc.gov/covid-data-tracker/COVIDData/getAjaxData?id=vaccination_demographics_data -o data/cdc_vaccinations_demographics.json
+        curl https://beta.healthdata.gov/api/views/di4u-7yu6/rows.csv?accessType=DOWNLOAD -o data/hhs/community_profile_counties_$(TZ=America/New_York date '+%Y%m%d').csv
```

Comment:
Quick change of plans here. Dave would prefer separate files for each day, so I'm generating a datestamp and moved this to the `hhs` directory.

Reply:
awesome! thank you :)